Assets/Scripts/Buddy/BuddyStats.cs

```diff
@@ -74,6 +74,13 @@ enum HappinessState
 		Happy,
 	}
 
+	enum BuddyState
+	{
+		Normal,
+		Dead,
+		Adult,
```

Comment:
I don't think that's needed in this case, I explicitly initialize the buddies to `Normal`.